bin/makepart

```diff
@@ -101,7 +101,7 @@ sed 's/#.*//;/^[[:blank:]]*$/d' \
 			rm -rf "$rootfs_work/overlay"
 		else
 			echo "overlay /var overlay defaults,lowerdir=/var,upperdir=/overlay/var/upper,workdir=/overlay/var/work,x-systemd.requires-mounts-for=/overlay 0 2" >> "$fstab"
```

Comment:
What about `/var`? Although we hadn't issues with something like the machine-id there, it may make sense to mount this directory as soon as possible, too. This way we keep the time for a read only /var short.